### PR TITLE
Restrictive permissions on filebeat sensitive files

### PR DESCRIPTION
**Pull request by Zenidd**
Hi team,

- This PR aims to set secure permissions for some filebeat related files and certs with too open permissions. These files use `0400` mask now: https://github.com/wazuh/wazuh-ansible/pull/382/commits/efcb55b52362b517b3b9343f0b2183d7a8ef149e

Greetings, JP Sáez

### CODE REVIEW

**Comment by Zenidd** [comment]
Requested changes to permissions regarding Kibana and ES are included in the following commits:

- https://github.com/wazuh/wazuh-ansible/pull/382/commits/eff4b38bedf6091786e26bf55c8ceef51f47cf61
- https://github.com/wazuh/wazuh-ansible/pull/382/commits/c6a3dda23ac56d0e35bc208586d1a7cb8ffa3af8

Also, syntax normalization to octal has been applied to every `mode` permission assignation:
- https://github.com/wazuh/wazuh-ansible/pull/382/commits/33fceff612a3f97c291c03158759b0ea0ad356f4

File: roles/elastic-stack/ansible-elasticsearch/tasks/xpack_security.yml [hunk 1]
```diff
@@ -149,6 +149,7 @@
   copy:
     src: "{{ item }}"
     dest: "{{ node_certs_destination }}/"
+    mode: 0444
   with_items:
```
Comment (jm404) [inline roles/elastic-stack/ansible-elasticsearch/tasks/xpack_security.yml]: Read all permissions should not be assigned to a .key afaik

File: roles/elastic-stack/ansible-kibana/tasks/main.yml [hunk 2]
```diff
@@ -28,6 +28,7 @@
   copy:
     src: "{{ item }}"
     dest: "{{ node_certs_destination }}/"
+    mode: 0444
   with_items:
     - "{{ master_certs_path }}/{{ kibana_node_name }}/{{ kibana_node_name }}.key"
```
Comment (jm404) [inline roles/elastic-stack/ansible-kibana/tasks/main.yml]: Read all permissions should not be assigned to a .key afaik

File: roles/elastic-stack/ansible-kibana/tasks/main.yml [hunk 3]
```diff
@@ -41,7 +42,7 @@
   copy:
     src: "{{ item }}"
     dest: "{{ node_certs_destination }}/"
-    mode: '0664'
+    mode: 0444
```
Comment (jm404) [inline roles/elastic-stack/ansible-kibana/tasks/main.yml]: Read all permissions should not be assigned to a .key afaik

File: roles/elastic-stack/ansible-kibana/tasks/main.yml [hunk 4]
```diff
@@ -78,7 +79,7 @@
     dest: /etc/kibana/kibana.yml
     owner: root
     group: root
-    mode: '0664'
+    mode: 0664
```
Comment (jm404) [inline roles/elastic-stack/ansible-kibana/tasks/main.yml]: Kibana.yml can contain credentials, remove the readall permissions.

Please verify if it should be `group:kibana`

File: roles/wazuh/ansible-filebeat/tasks/main.yml [hunk 5]
```diff
@@ -30,6 +30,7 @@
   copy:
     src: "{{ item }}"
     dest: "{{ node_certs_destination }}/"
+    mode: '0444'
```
Comment (jm404) [inline roles/wazuh/ansible-filebeat/tasks/main.yml]: Read all permissions should not be assigned to a .key afaik

File: roles/wazuh/ansible-filebeat/tasks/main.yml [hunk 6]
```diff
@@ -43,6 +44,7 @@
...
```
Comment (jm404) [inline roles/wazuh/ansible-filebeat/tasks/main.yml]: Read all permissions should not be assigned to a .key afaik

File: roles/elastic-stack/ansible-elasticsearch/tasks/xpack_security.yml (hunk 1 above)
Comment (Zenidd) [reply roles/elastic-stack/ansible-elasticsearch/tasks/xpack_security.yml]: Hello, @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05

File: roles/elastic-stack/ansible-kibana/tasks/main.yml (hunk 2 above)
Comment (Zenidd) [reply roles/elastic-stack/ansible-kibana/tasks/main.yml]: Hello @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05

File: roles/elastic-stack/ansible-kibana/tasks/main.yml (hunk 3 above)
Comment (Zenidd) [reply roles/elastic-stack/ansible-kibana/tasks/main.yml]: Hello @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05

File: roles/wazuh/ansible-filebeat/tasks/main.yml (hunk 5 above)
Comment (Zenidd) [reply roles/wazuh/ansible-filebeat/tasks/main.yml]: Hello @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05

File: roles/wazuh/ansible-filebeat/tasks/main.yml (hunk 6 above)
Comment (Zenidd) [reply roles/wazuh/ansible-filebeat/tasks/main.yml]: Hello @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05

File: roles/elastic-stack/ansible-kibana/tasks/main.yml (hunk 4 above)
Comment (Zenidd) [reply roles/elastic-stack/ansible-kibana/tasks/main.yml]: image

Comment (Zenidd) [reply roles/elastic-stack/ansible-kibana/tasks/main.yml]: Default kibana permissions and owner:group after fresh repo install

File: roles/wazuh/ansible-filebeat/tasks/main.yml (hunk 6 above)
Comment (Zenidd) [reply roles/wazuh/ansible-filebeat/tasks/main.yml]: Hello @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05

**Pull request merged**